index.html

```diff
@@ -17,9 +17,9 @@
               <br />
               <a href="#Introduction" class="nav-link">Introduction</a>
               <br />
-              <a href="#What-you-should-already-know" class="nav-link"
-                >What you should already know</a
-              >
+              <a href="#What-you-should-already-know" class="nav-link">
+                What you should already know
```

Comment:
- Please make use of inline comments to request a change, as it is better for the student to see what exact place where changes are required. [Tip]  have at least two or more inline comments requesting changes.

Reply:
- Please make your request polite while asking for changes to be made, use words like please, kindly change, etc.

You can reach out to me on slack if you find this review confusing `@kanu mike chibundu`